v2/sunshine/notely_app/assets/user_sidebar.html

```diff
@@ -49,23 +49,37 @@
           function recordActivity() {
             var note = document.getElementById('note');
             var type = document.getElementById('type');
-            var description = '[' + type.value + '] ' + note.value;
+            var description = '[' + type.text + '] ' + note.value;
```

Comment:
Just curious, did you modify this from `value` to `text` so that we have an title-cased event type?

Reply:
It was - so that the description would present nicely. I have changed this now to be explicit (afaik there isn't a titlecase or similar in javascript)